releasenotes/notes/feat-llmobs-vertexai-f58488859472c7b5.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
```

Comment:
Can we also add this as a follow up item to update our public integrations doc to include vertexAI? https://docs.datadoghq.com/llm_observability/setup/auto_instrumentation?tab=python